src/runtime_src/driver/xclng/tools/xbutil/xbutil.cpp

```diff
@@ -549,16 +553,19 @@ int main(int argc, char *argv[])
 
     unsigned int total = pcidev::get_dev_total();
     unsigned int count = pcidev::get_dev_ready();
-    if (total == 0) {
-        std::cout << "ERROR: No card found\n";
-        return 1;
-    }
+
     if (cmd != xcldev::DUMP)
         std::cout << "INFO: Found total " << total << " card(s), "
                   << count << " are usable" << std::endl;
 
+    if ((cmd == xcldev::QUERY) || (cmd == xcldev::SCAN))
```

Comment:
Instead of adding lines 561 and 562, would it be the same to move L562 after L567 and inside xbutil::dump() or before dump() copy to xbutil.cpp:L763?

Reply:
We need to provide this information even if no device is present